tensorflow/tsl/platform/cpu_info.cc

```diff
@@ -438,7 +465,7 @@ void InitCPUIDInfo() {
   absl::call_once(cpuid_once_flag, CPUIDInfo::Initialize);
 }
 
-#endif
+#endif  // PLATFORM_IS_ARM64
```

Comment:
This will be changed as well. But I think I'll just merge conflicts internally.
```suggestion
#endif  // PLATFORM_IS_ARM64 && !__APPLE__ && !__OpenBSD__
```

Reply:
I will have to add these again I think. By mistake I re-wrote the branch history. Sorry

Reply:
No worries, I've added that internally. :)